examples/react-alt/readme.md

```diff
@@ -0,0 +1,45 @@
+# React + Alt TodoMVC Example
+
+> React is a JavaScript library for creating user interfaces. Its core principles are declarative code, efficiency, and flexibility. Simply specify what your component looks like and React will keep it up-to-date when the underlying data changes. Alt is one of many implementations of Facebook's Flux architecture.
```

Comment:
There's some stuff here: http://alt.js.org/

You can also check this out: https://github.com/goatslacker/alt#why-you-should-be-using-alt